func-complete-check/check-validator/check-validator.v1.ts

```diff
@@ -38,18 +34,19 @@ export class CheckValidatorV1 implements ICheckValidator {
     // this should fail outside of the catch as we wont be able to update the entity
     // without a reference to it and should rightly go on the dead letter queue
     const receivedCheck = this.findReceivedCheck(functionBindings.receivedCheckTable)
+    let checkData
     try {
       this.detectArchive(receivedCheck)
       const decompressedString = this._compressionService.decompress(receivedCheck.archive)
-      const checkData = JSON.parse(decompressedString)
+      checkData = JSON.parse(decompressedString)
       this.validateCheckStructure(checkData)
     } catch (error) {
       await this.setReceivedCheckAsInvalid(error.message, receivedCheck)
       logger.error(error.message)
       return
     }
 
-    await this.setReceivedCheckAsValid(receivedCheck)
+    await this.setReceivedCheckAsValid(receivedCheck, checkData)
     logger.info(`receivedCheck successfully validated`)
```

Comment:
This message will be useless in production without a checkCode.  It might be better at `debug` level or removed.